pkg/kubectl/cmd/create.go

```diff
@@ -285,6 +285,28 @@ func NameFromCommandArgs(cmd *cobra.Command, args []string) (string, error) {
 	return args[0], nil
 }
 
+// PortsFromCommandArgs is a function to gather ports config from args. It supports deprecating "--tcp" arg.
+func PortsFromCommandArgs(cmd *cobra.Command) ([]string, error) {
```

Comment:
I'd prefer we avoid exporting methods unless we explicitly need them to be exported.